cmd/gcp-controller-manager/app/gke_approver_options.go

```diff
@@ -0,0 +1,111 @@
+package app
+
+import (
+	"context"
+	"crypto/x509"
+	"fmt"
+	"sort"
+	"strings"
+
+	"cloud.google.com/go/compute/metadata"
+	"golang.org/x/oauth2"
+	compute "google.golang.org/api/compute/v1"
+	container "google.golang.org/api/container/v1"
+	gcfg "gopkg.in/gcfg.v1"
+	warnings "gopkg.in/warnings.v0"
+	"k8s.io/kubernetes/pkg/cloudprovider/providers/gce"
+)
+
+type gcpConfig struct {
```

Comment:
We use this in the annotater too. These aren't all just approver options.

Reply:
Moved to options.go